lang/proto/proto.py

```diff
@@ -0,0 +1,37 @@
+from talon import Module, Context, actions, ui, imgui, settings
+
+mod = Module()
+
+ctx = Context()
+ctx.matches = r'''
+mode: user.protobuf
+mode: user.auto_lang
+and code.language: protobuf
+'''
+
+ctx.lists['user.code_type'] = {
+    'string': 'string',
+    'bytes': 'bytes',
+    'you 64': 'uint64',
```

Comment:
Numbers in spoken forms don't work, you'll need to spell them out eg "you sixty four".